ci/run_autoscaling_osp18.yml

```diff
@@ -60,14 +61,49 @@
         - name: "Run Telemetry Autoscaling tests"
           ansible.builtin.import_role:
             name: telemetry_autoscaling
-          
-      always:   
+ 
+        - ansible.builtin.shell:
+            cmd: |
+              cat ./*.xml ~/*.xml
+            chdir: "{{ fvt_dir }}"
+          register: output
+          ignore_errors: true
+
+        - ansible.builtin.shell: cp {{ fvt_dir }}/*.xml {{ logs_dir }}/ && cp ~/*.xml {{ logs_dir }}/
+          ignore_errors: true
+
+        - name: Save ansible output to a file
+          ansible.builtin.copy:
+            content: "{{ output.stdout }}"
+            dest: "{{ logs_dir }}/autoscaling/ansible_run.log"
+
+      always:
         - name: Revert the version update
           ansible.builtin.shell:
             cmd: |
               oc patch  openstackversions controlplane  --type json -p='[{"op": "replace", "path": "/spec/customContainerImages", "value": {} }]'
           when: "{{ patch_openstackversions | bool }}"
 
+        - name: "Where is that XML file!?"
```

Comment:
I _JUST_ realised why this isn't working. 

The junit callback create the output file only at the end of the playbook execution.

We need to run the report result as a a separate hook AFTER  the other tests are run.

Reply:
Technically, report result can stay where it is. We just need to have a post_test hook that runs to do the copying of the data.
This should be temporary, since the output_dir can be configured.